vars/capsuleSystemTests.groovy

```diff
@@ -323,6 +321,79 @@ void call(Map additionalConfig=[:], parametersOverride=[:]) {
           }
         }
       }
+
+      stage('protocol upgrade proposal step') {
+        when {
+          expression {
+            params.RUN_PROTOCOL_UPGRADE_PROPOSAL
+          }
+        }
+        environment {
+          PATH = "${networkPath}:${env.PATH}"
+        }
+        options {
+          timeout(time: 10, unit: 'MINUTES')
+        }
+
+        steps {
+          script {
+            int upgradeProposalOffset = 100
+            def getLastBlock = { boolean silent ->
+              return vegautils.shellOutput('''devopsscripts vegacapsule last-block \
+                  --output value-only \
+                  --network-home-path ''' + testNetworkDir + '''/testnet \
+                  --local
+                ''', silent).toInteger()
+            }
+            int initNetworkHeight = getLastBlock(false)
+            int proposalBlock = initNetworkHeight + upgradeProposalOffset
+            print('Current network heigh is ' + initNetworkHeight)
+            print('Proposing protocol upgrade on block ' + proposalBlock)
+            sh '''vegacapsule nodes protocol-upgrade \
+                --propose \
+                --home-path ''' + testNetworkDir + '''/testnet \
+                --template-path system-tests/vegacapsule/net_configs/visor_run.tmpl \
+                --height ''' + proposalBlock + ''' \
+                --release-tag v0.58.0
```

Comment:
hardcoded `v0.58.0`

Reply:
at the moment @ze97286 mentioned this needs to be just valid tak because we do not do real upgrade, just needs to propose upgrade to make sure network is not producing blocks any more